eahub/localgroups/models.py

```diff
@@ -32,6 +32,7 @@ class LocalGroup(models.Model):
     facebook_page = models.URLField(blank=True)
     email = models.EmailField(blank=True)
     meetup_url = models.URLField(blank=True)
+    active = models.BooleanField(default=True)
```

Comment:
Would prefer for this to be named `is_active` so it's clearer that it's a boolean. This change will necessitate redoing the migrations.

Also, have you confirmed that it's possible to create a new inactive group?

Reply:
> Would prefer for this to be named is_active so it's clearer that it's a boolean. This change will > necessitate redoing the migrations.

Tbh, don't think it's worth the time. I think it should be clear to anyone who reads this.

> Also, have you confirmed that it's possible to create a new inactive group?

No. Why is this relevant?